thirdeye/thirdeye-frontend/app/api/self-serve.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * GET request for all alert subscription groups
+ * @see {@link namepathOrURL|link text}
```

Comment:
These links will be replaced once we know where the swagger doc will live.

Reply:
I like how we will link it to Swagger. That is very cool indeed!